requirements/base.txt

```diff
@@ -10,3 +10,9 @@ tornado>=4.2.1,<5.0; python_version >= '3.4'
 
 # Required by Tornado to handle threads stuff.
 futures>=2.0; python_version < '3.0'
+
+# Available template libraries that can be used
+Genshi
```

Comment:
Nope. They are accurate.
Now that we actually have more hook in our pre commit config, we can even target the requirements generation by `pre-commit run -av pip-tools-compile`.

Reply:
These tests aren't linux-specific, is `linux.in` still the right place or would I add them to all the `*.in` files?

Reply:
Please add them to Linux, windows and Darwin `.in` files.

Reply:
Gotcha. Thanks y'all for answering my question and allowing me to stop hot-patching these PRs into my own fork.

Reply:
You know that these requirements are for testing right?
They won't be part of Salt's hard dependencies.
Just checking.

Reply:
yeah i know. my first commits didn't actually modify these requirements, i only had to add them because i was required to write testcases even for the templates that I didn't fix. If you look at my first commit, i literally only modified `render_cheetah_templ`. Everything else is a result of this need for testcases.